misc/ide.talon

```diff
@@ -15,4 +15,14 @@ extract method: user.extract_method()
 refactor in line: user.refactor_in_line()
 refactor move: user.refactor_move()
 refactor rename: user.refactor_rename()
-rename file: user.rename_file()
\ No newline at end of file
+rename file: user.rename_file()
+
```

Comment:
I thought it made more sense to keep this context free so each IDE can specify it's own context, otherwise you'd duplicate the context (as this for would contain every use then each implementation would have to specify which IDE it is). Talon won't do anything if it can't find an implementation for the context so it won't fire unexpectedly (is my understanding at least).

Reply:
I see this one kinda like generic_browser, maybe -
https://github.com/knausj85/knausj_talon/blob/master/apps/generic_browser.talon

so as we add support for various IDEs, we'd add the appropriate app name. long term, we'd tag the e.g. jetbrains context with 
tag: ide
-

or similar. Long term, tags get rid of any duplication (same applies to generic_browser).

Thoughts?

Reply:
Tags sound good, when they get here. Having all the app names in one place is a quick way to see which ide's are supported and I guess we can wait until it becomes too long and unweild-y to manage before optimising it :)